terraform/aws/env/us-east/variables.tf

```diff
@@ -11,19 +11,25 @@ variable "region" {
   default     = "us-east-1"
 }
 
+variable "USE_HCP_PACKER"{
+  default     = true
+}
+
 variable "ami" {
+  default ="12345"
 }
 
 variable "server_instance_type" {
   description = "The AWS instance type to use for servers."
-  default     = "t2.medium"
+  default     = "t2.micro"
 }
 
 variable "client_instance_type" {
   description = "The AWS instance type to use for clients."
-  default     = "t2.medium"
+  default     = "t2.micro"
```

Comment:
I'd recommend against changing this. A medium is frankly pretty tight as it is for Nomad.